Add more information for running individual unit tests (#4518)
qgis/QGIS-Documentation: source/docs/developers_guide/unittesting.rst

  ## 0 tests passed, 1 tests failed out of 1

>   The following tests FAILED./output/bin/qgis_dxfexporttest:

Author: ```suggestion
  The following tests FAILED:
```

  ## 1- qgis_applicationtest (Failed)
  Errors while running CTest

Running individual tests
------------------------

C++ tests are ordinary applications. You can run them from the build folder
like any executable.

.. code-block:: bash

  $ ./output/bin/qgis_dxfexporttest
  
  ********* Start testing of TestQgsDxfExport *********
  Config: Using QtTest library 5.12.5, Qt 5.12.5 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 9.2.1 20190827 (Red Hat 9.2.1-1))
  PASS   : TestQgsDxfExport::initTestCase()
  PASS   : TestQgsDxfExport::testPoints()
  PASS   : TestQgsDxfExport::testLines()
  ...
  Totals: 19 passed, 4 failed, 0 skipped, 0 blacklisted, 612ms
  ********* Finished testing of TestQgsDxfExport *********

These tests also take `command line arguments

> <https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_,

Reviewer: ```suggestion
<https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_.
```

> <https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_,

Reviewer: ```suggestion
<https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_.
```

> this allows to run a specific subset of tests.

Reviewer: ```suggestion
This makes it possible to run a specific subset of tests:
```

> this allows to run a specific subset of tests.

Reviewer: ```suggestion
This makes it possible to run a specific subset of tests:
```

.. code-block:: bash

  $ ./output/bin/qgis_dxfexporttest testPoints   
  ********* Start testing of TestQgsDxfExport *********
  Config: Using QtTest library 5.12.5, Qt 5.12.5 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 9.2.1 20190827 (Red Hat 9.2.1-1))
  PASS   : TestQgsDxfExport::initTestCase()
  PASS   : TestQgsDxfExport::testPoints()
  PASS   : TestQgsDxfExport::cleanupTestCase()
  Totals: 3 passed, 0 failed, 0 skipped, 0 blacklisted, 272ms
  ********* Finished testing of TestQgsDxfExport *********

Debugging unit tests
--------------------

C++ Tests
.........

For C++ unit tests, QtCreator automatically adds run targets, so you can start
them from the debugger.

If you go to *Projects* and there to the *Build & Run* --> Desktop *Run* tab, you can

> also specify command line parameters that will allow to run only a subset of the tests

Reviewer: ```suggestion
also specify command line parameters that will allow a subset of the tests to be run
```
